deltacat/compute/compactor/model/compactor_version.py

```diff
@@ -0,0 +1,6 @@
+from enum import Enum
+
+
+class CompactorVersion(Enum):
```

Comment:
Can we make this a `(str, Enum)` and then we can make comparisons without `.value` method

Reply:
Good point.